<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTimer</class>
 <widget class="QDialog" name="DialogTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogTimer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhPreferNumbers</set>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTimer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTimer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>DialogTimer</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPause</sender>
   <signal>clicked()</signal>
   <receiver>DialogTimer</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogTimer</receiver>
   <slot>textChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
  <slot>pause()</slot>
  <slot>textChanged()</slot>
 </slots>
</ui>
